packages/docs/src/components/tests/DocsCode.spec.js

```diff
@@ -0,0 +1,50 @@
+import { shallowMount } from '@vue/test-utils'
+import DocsCode from '../DocsCode.vue'
+import PrismWrapper from '../PrismWrapper.vue'
+
+describe('DocsCode', () => {
+  let wrapper
+
+  const createComponent = ({ props } = {}) => {
+    wrapper = shallowMount(DocsCode, {
+      props,
+      global: {
+        mocks: {
+          $t: (msg) => msg,
+        },
+      },
+    })
+  }
+
+  afterEach(() => {
+    wrapper.unmount()
+    wrapper = null
+  })
+
+  it('sanity', () => {
+    createComponent()
+
+    expect(wrapper.exists()).toBe(true)
+    expect(wrapper.findComponent(PrismWrapper).exists()).toBe(true)
+  })
+
+  it('apply translations for `code` prop', () => {
+    const ORIGIN_CODE = `<template>
+      <div>
+        <some-component :prop="$t('passed prop')" />
+        <some-component>{{ $t('passed slot') }}</some-component>
+      </div>
+    </template>`
+
+    const OUTPUT_CODE = `<template>
+      <div>
+        <some-component :prop="'passed prop'" />
+        <some-component>{{ 'passed slot' }}</some-component>
```

Comment:
possible to have `passed slot` in stead of `{{ 'passed slot' }}`? We discussed this before.